vignettes/getting_model_fits.Rmd

```diff
@@ -0,0 +1,140 @@
+---

```

Comment:
👍

Looks great. I might, at some point, try and add a section about subbing in differnt growth models via misc, but that can wait for now!